package.json

```diff
@@ -37,7 +37,7 @@
         "aperture-config": "git+https://github.com/joyent/aperture-config.git#master",
         "joyent-schemas": "git+https://github.com/joyent/schemas.git#caf3a226ed0707f5da897e1da151cc6d97fccda2",
         "jsprim": "0.6.1",
-        "verror": "1.6.1"
+        "verror": "1.10.0"
```

Comment:
yep, it's for hasCauseWithName which came in 1.9.0. I figured might as well go with latest.